docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1221,17 +1213,159 @@ paths:
             $ref: '#/definitions/Problem'
         '409':
           description: |
-            Conflict. There are no empty slots for this subscriptions. The amount of consumers for this subscription
-            already equals the maximal value - the total amount of this subscription partitions.
+            Conflict. There are several possible reasons for receiving this status code:
+
+            - There are no empty slots for this subscriptions. The amount of consumers for this subscription
+            already equals the maximum value - the total number of partitions in this subscription.
+
+            - Request to reset subscription cursors is still in progress.
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Access is forbidden for the client or event type
+          schema:
+            $ref: '#/definitions/Problem'
+
+    post:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        GET with body.
+        Starts a new stream for reading events from this subscription. Using this endpoint it is possible to specify
+        which partitions to read in this stream.
+
+        It is possible to specify a partition that is already consumed by another stream. In this case, there can be
+        two outcomes, depending on how the current assignment was obtained. (1) If the partition is already assigned to
+        another stream through automatic assignment, it will be possible to reassign it to new stream, and the request
+        will succeed. (2) If the partition is already assigned to another stream because of a request to stream from
+        this partition specifically, then it will not be possible to reasiign the partition to new stream, and the
+        request will fail.
+
+        It is only possible to specify partitions that belong to this subscription.
+
+        When using this endpoint, the balancing of partitions is the user's responsibility. It is also the user's
+        responsibility to detect, and react to, changes in the number of partitions in the subscription (following the
+        re-partitioning of an event type). Using the GET /subscriptions/{subscription_id}/stats endpoint can be helpful.
+
+        The rest of the behavior is identical to endpoint GET /subscriptions/{subscription_id}/events.
+      parameters:
+        - name: streamParameters
```

Comment:
@v-stepanov Hi, @tfrauenstein and I have talked about this API change. We would suggest to keep the `get-with-body` as symmetric to the `get` as possible. Therefore, we would suggest to reuse the query parameter of the `get` instead of appending every parameter to the body. We think that this would be more to the expectations of the consumers and help them easier to adapt to the new endpoint.

Reply:
@tkrop 
In terms of adoption - I can not fully agree with you. One will have to change only:
 - method to call
 - append non-empty body with specific parameter
 - check that library that is being used for http interaction will actually put parameters to a query string, instead of post body (or will not miss it, because post body is already there).

Probably in most cases it will work without a lot of modifications (but because of uncertainty will not simplify onboarding). 

From the other hand it will produce api, that is not well-looking, and we will not be able to get rid of it. Not well looking is because all the parameters that are passed to streaming endpoint are on the same level of hierarchy - it is all streaming parameters. The reason of putting them to different places is unclear.

Also, in case if we are talking about simplification, we should (of course) think about unification - only one of this methods will be used by particular client (I hope), and for this particular client using unified way of specifying parameters is actually the simplest approach, meaning simplification of onboarding.